src/api/users.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line
+const API_URL = `https://mate-api.herokuapp.com/`;
+
+export function getAll(url, options) {
+  return fetch(
+    !options ? `${API_URL}${url}` : `${API_URL}${url}${options}`,
+  )
```

Comment:
no need to check for options

```suggestion
  return fetch(`${API_URL}${url}${options}`)
```